diff --git a/target/linux/ipq807x/files-5.10/arch/arm64/boot/dts/qcom/ipq8074-ac-cpu.dtsi b/target/linux/ipq807x/files-5.10/arch/arm64/boot/dts/qcom/ipq8074-ac-cpu.dtsi
index 4b2c3c484..bc7efdd0e 100644
--- a/arch/arm64/boot/dts/qcom/ipq8074-ac-cpu.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq8074-ac-cpu.dtsi
@@ -32,12 +32,37 @@
 		opp-1017600000 {
 			opp-hz = /bits/ 64 <1017600000>;
 			opp-microvolt = <704000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-1382400000 {
 			opp-hz = /bits/ 64 <1382400000>;
-			opp-microvolt = <824000>;
-			clock-latency-ns = <200000>;
+			opp-microvolt = <784000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-1651200000 {
+			opp-hz = /bits/ 64 <1651200000>;
+			opp-microvolt = <832000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-1843200000 {
+			opp-hz = /bits/ 64 <1843200000>;
+			opp-microvolt = <880000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-1920000000 {
+			opp-hz = /bits/ 64 <1920000000>;
+			opp-microvolt = <904000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-2208000000 {
+			opp-hz = /bits/ 64 <2208000000>;
+			opp-microvolt = <984000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-2496000000 {
+			opp-hz = /bits/ 64 <2496000000>;
+			opp-microvolt = <1040000>;
+			clock-latency-ns = <100000>;
 		};
 	};
 };
diff --git a/target/linux/ipq807x/files-5.10/arch/arm64/boot/dts/qcom/ipq8074-hk-cpu.dtsi b/target/linux/ipq807x/files-5.10/arch/arm64/boot/dts/qcom/ipq8074-hk-cpu.dtsi
index be7b4a972..3c2e70da3 100644
--- a/arch/arm64/boot/dts/qcom/ipq8074-hk-cpu.dtsi
+++ b/arch/arm64/boot/dts/qcom/ipq8074-hk-cpu.dtsi
@@ -32,32 +32,37 @@
 		opp-1017600000 {
 			opp-hz = /bits/ 64 <1017600000>;
 			opp-microvolt = <704000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-1382400000 {
 			opp-hz = /bits/ 64 <1382400000>;
 			opp-microvolt = <784000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-1651200000 {
 			opp-hz = /bits/ 64 <1651200000>;
 			opp-microvolt = <832000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-1843200000 {
 			opp-hz = /bits/ 64 <1843200000>;
 			opp-microvolt = <880000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-1920000000 {
 			opp-hz = /bits/ 64 <1920000000>;
 			opp-microvolt = <904000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
 		};
 		opp-2208000000 {
 			opp-hz = /bits/ 64 <2208000000>;
 			opp-microvolt = <984000>;
-			clock-latency-ns = <200000>;
+			clock-latency-ns = <100000>;
+		};
+		opp-2496000000 {
+			opp-hz = /bits/ 64 <2496000000>;
+			opp-microvolt = <1040000>;
+			clock-latency-ns = <100000>;
 		};
 	};
 };
